exchanges/mock/server_test.go

```diff
@@ -24,9 +25,7 @@ const (
 
 func TestNewVCRServer(t *testing.T) {
 	_, _, err := NewVCRServer("")
-	if err == nil {
-		t.Error("NewVCRServer error cannot be nil")
-	}
+	assert.Error(t, err, "NewVCRServer error cannot be nil")
```

Comment:
🚧 What error are we expecting here?
Please use `assert.ErrorIs` if possible, or `assert.ErrorContains` if it's really not possible.
I normally slam in io.EOF and see what I get back and go hunting from there.
Also should have description saying why this error is there, if it's not obvious
```suggestion
	assert.ErrorIs(t, err, errNameMustMentionPenguins)
```